modules/mod_ginger_rdf/dispatch/dispatch

```diff
@@ -1,4 +1,5 @@
 [
     {rsc_json_ld, ["rdf", id], controller_rdf, [{serialization, json_ld}, {ontology, schema_org}]},
-    {rsc_turtle, ["turtle", id], controller_rdf, [{serialization, turtle}, {ontology, schema_org}]}
+    {rsc_turtle, ["turtle", id], controller_rdf, [{serialization, turtle}, {ontology, schema_org}]},
+    {hydra, ["hydra", id], controller_hydra, [{ontology, schema_org}]}
```

Comment:
If one would like to be able to include multiple ontologies (which might be completely out of scope for the current use case), would making the ontology parameter a list complicate things too much?

Reply:
Done in #697.